benchmarking/mod/benchmark.py

```diff
@@ -359,7 +359,7 @@ def testjob_distribution(self, disk_num_per_client, instance_list):
         client_num = 0
         self.cluster["testjob_distribution"] = {}
         for client in self.cluster["client"]:
-            vclient_total = int(disk_num_per_client[client_num])
+            vclient_total = int(disk_num_per_client[client_num]) if client_num < len(disk_num_per_client) else int(disk_num_per_client[-1])
```

Comment:
I am thinking to make this code in config level, to check if on specify one number in disk_num_per_client, and make as client-number list in configuration.
Not hack in codes